events/quarkus/src/test/java/org/projectnessie/events/quarkus/TestQuarkusEventsWithTracing.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (C) 2023 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.events.quarkus;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.junit.TestProfile;
+import java.util.HashMap;
+import java.util.Map;
+import javax.inject.Inject;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.projectnessie.events.quarkus.assertions.EventAssertions;
+import org.projectnessie.events.quarkus.assertions.MetricsAssertions;
+import org.projectnessie.events.quarkus.assertions.TracingAssertions;
+import org.projectnessie.events.quarkus.scenarios.EventScenarios;
+
+@QuarkusTest
+@TestProfile(TestQuarkusEventsWithTracing.Profile.class)
+class TestQuarkusEventsWithTracing {
+
+  @Inject EventScenarios scenarios;
+  @Inject EventAssertions events;
+  @Inject TracingAssertions tracing;
+  @Inject MetricsAssertions metrics;
+
+  @AfterEach
+  void reset() {
+    events.reset();
+    tracing.reset();
+    metrics.reset();
+  }
+
+  @Test
+  void testCommitWithTracing() {
```

Comment:
Here is a screenshot of Jaeger UI for the spans produced by this test:

<img width="1947" alt="Screenshot 2023-05-22 at 15 02 24" src="https://github.com/projectnessie/nessie/assets/463876/a4c598e5-2a1f-4fa7-9f1f-c550f3d76e46">